proto/os/types/v1/indexer.proto

```diff
@@ -0,0 +1,32 @@
+// Copyright Tharsis Labs Ltd.(Evmos)
+// SPDX-License-Identifier:ENCL-1.0(https://github.com/evmos/evmos/blob/main/LICENSE)
```

Comment:
I would create a new LICENSE file in this repo and point this link to that one tbh

Reply:
as discussed on Slack, will adjust on a separate PR for all files